src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
 public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[index].name();
```

Comment:
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```

Reply:
Please, follow the comments. If you have some questions, please ask. 

You should create class-leve variable Random, to avoid creating it each time, when you call getRandomColor() method. 

I left this comment 
`private final Random random = new Random(); `

And in method getRandomColor() you just create `int index = random.nextInt(Color.values().length); `